modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -24,19 +25,24 @@ import (
 	"strings"
 	"time"
 
-	"github.com/caddyserver/caddy/v2"
-	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
 	"github.com/dustin/go-humanize"
 	"go.uber.org/zap"
+
+	"github.com/caddyserver/caddy/v2"
```

Comment:
Do you guys have a editorconfig for JetBrains Goland? 

My IDE is set to group imports by their source (internal / vendor)  
that's why it reorders these.

Should I revert?